resources/js/Utils/formatter.ts

```diff
@@ -1,2 +1,5 @@
 export const itemFormatter = (items: any[]) =>
     items.map((item: any) => item.name).join(', ')
+
+export const characterNameFormatter = (characters: any[]) =>
+    characters.map((character: any) => character.name).join(', ')
```

Comment:
if these are the same as itemFormatter, I think you can write a more generic function to accommodate both

Reply:
You're right, Ill join these two and update it in both the Campaign table and Item table